Userland/Applications/Run/RunWindow.cpp

```diff
@@ -168,3 +180,35 @@ bool RunWindow::run_via_launch(const String& run_input)
 
     return true;
 }
+
+String RunWindow::history_file_path()
+{
+    return AK::LexicalPath::canonicalized_path(String::formatted("{}/{}", Core::StandardPaths::config_directory(), "RunHistory.txt"));
```

Comment:
No need to qualify things with the `AK::` prefix

Reply:
Not sure why it never dawned on me that _everything_ in `AK` is in the global namespace, considering how often I use `String`s and `Vector`s without qualifying the namespace 😄 Will give everything a look-over tonight (7/8 hrs or so) 👍